code/modules/reagents/reagent_containers/patch.dm

```diff
@@ -11,7 +11,13 @@
 
 /obj/item/weapon/reagent_containers/pill/patch/New()
 	..()
-	icon_state = "bandaid" // thanks inheritance
+	// thanks inheritance
+	if(name == "brute patch")
+		icon_state = "bandaid_brute"
+	else if (name == "burn patch")
+		icon_state = "bandaid_burn"
+	else
+		icon_state = "bandaid_generic"
```

Comment:
Uh, wouldn't it make more sense to override them in the `/obj/item/weapon/reagent_containers/pill/patch/styptic` and `/obj/item/weapon/reagent_containers/pill/patch/silver_sulf` right below, instead of relying on `name` for the logic? Something like this:
```
/obj/item/weapon/reagent_containers/pill/patch/styptic/New()
	..()
	icon_state = "bandaid_brute" 
```
Since it's setting icon_state after calling its parent proc on ..(), it shouldn't cause issues.